lib/system.nim

```diff
@@ -4073,6 +4078,25 @@ template closureScope*(body: untyped): untyped =
   ##   myClosure() # outputs 3
   (proc() = body)()
 
+template once*(body: untyped): untyped =
+  ## Executes a block of code only once (the first time the block is reached).
+  ## When code hot reloading is enabled, protects top-level code from being
```

Comment:
Minor nitpick but please s/code hot reloading/hot code reloading/

Reply:
Hrm, you are right after googlefighting the terms. I'l change that.